src/hb-ot-shape-complex-arabic-fallback.hh

```diff
@@ -170,7 +170,7 @@ arabic_fallback_synthesize_lookup_ligature (const hb_ot_shape_plan_t *plan HB_UN
   c.end_serialize ();
   /* TODO sanitize the results? */
 
-  return ret ? c.copy<OT::SubstLookup> () : nullptr;
```

Comment:
The one that fuzzer has discovered is this one but the other has the same code so anything happens here should happen this also most likely.